bundle/compliance/lib/common/common.rego

```diff
@@ -5,6 +5,10 @@ metadata = {"opa_version": opa_version}
 # get OPA version
 opa_version := opa.runtime().version
 
+has_key(x, k) {
+	x[k]
+}
+
```

Comment:
we can keep it as it makes the check clearer, but checking if key exists using `x[k]` is a basic feature in rego

Reply:
removed